src/Tasks/Resources/Strings.resx

```diff
@@ -1170,6 +1170,9 @@
     <comment>{StrBegin="MSB3825: "}</comment>
   </data>
 
+  <data name="GenerateResource.MimeTypeNotSupportedOnCore">
+    <value>Resource \"{0}\" in \"{1}\"uses MIME type \"{2}\", which is not supported by MSBuild when running on .NET.</value>
```

Comment:
Do you think I should give this an MSB error code? It didn't have one before, so I left it that way, but it seems like a reasonable candidate.